File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        
        s = s.toLowerCase();
        int p = 0;
        int y = 0;
        
        for (int i = 0; i < s.length(); i++) {
            
            if (s.charAt(i) == 'p') p++;
            if (s.charAt(i) == 'y') y++;
        }
        
        if ((p == 0 && y == 0) || p == y) return true;
        
        return false;
    }
}

File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    s = s.lower()
    if 'p' not in s and 'y' not in s:
        return True
    p_num = 0
    y_num = 0
    for i in s:
        if i == 'p':
            p_num += 1
        elif i == 'y':
            y_num += 1
    return p_num == y_num